torch/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ else()
   project(torch CXX C)
   find_package(Caffe2 REQUIRED)
   option(USE_CUDA "Use CUDA" ON)
+  set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
Why do we need this two places?

Reply:
It seems we allows torch/CMakeLists.txt to be called standalone.